app/accelerators.js

```diff
@@ -11,38 +11,38 @@ const applicationMenu = { // app/menu.js
   // Shell/File menu
   newWindow: 'N',
   newTab: 'T',
-  splitVertically: isMac ? 'D' : 'Shift+E',
-  splitHorizontally: isMac ? 'Shift+D' : 'Shift+O',
+  splitVertically: 'D',
+  splitHorizontally: isMac ? 'Shift+D' : 'E',
   closeSession: 'W',
-  closeWindow: 'Shift+W',
+  closeWindow: isMac ? 'Shift+W' : 'Q',
 
   // Edit menu
   undo: 'Z',
-  redo: 'Shift+Z',
+  redo: isMac ? 'Shift+Z' : 'Y',
   cut: 'X',
-  copy: isMac ? 'C' : 'Shift+C',
+  copy: 'C',
   paste: 'V',
   selectAll: 'A',
   clear: 'K',
   emojis: isMac ? 'Ctrl+Cmd+Space' : '',
 
   // View menu
   reload: 'R',
-  fullReload: 'Shift+R',
-  toggleDevTools: isMac ? 'Alt+I' : 'Shift+I',
+  fullReload: isMac ? 'Shift+R' : 'F5',
+  toggleDevTools: isMac ? 'Alt+I' : 'I',
   resetZoom: '0',
   zoomIn: 'plus',
   zoomOut: '-',
 
   // Plugins menu
-  updatePlugins: 'Shift+U',
+  updatePlugins: isMac ? 'Shift+U' : 'U',
 
   // Window menu
   minimize: 'M',
-  showPreviousTab: 'Alt+Left',
-  showNextTab: 'Alt+Right',
+  showPreviousTab: isMac ? 'Alt+Left' : '+Ctrl+PageDown',
+  showNextTab: isMac ? 'Alt+Right' : '+Ctrl+PageUp',
   selectNextPane: 'Ctrl+Alt+Tab',
```

Comment:
just throwing out some ideas.. in atom, the panes are indexed, and we can switch between them using `ctrl+[index]`.. would it be possible to have something similar?